packages/zowe-explorer-api/src/tree/ZoweTreeNode.ts

```diff
@@ -76,8 +77,17 @@ export class ZoweTreeNode extends vscode.TreeItem {
      * @returns {imperative.IProfileLoaded}
      */
     public getProfile(profilesCache?: ProfilesCache): imperative.IProfileLoaded {
-        if (this.profile != null && profilesCache?.loadNamedProfile != null) {
-            return profilesCache.loadNamedProfile(this.profile.name);
+        if (this.profile != null && profilesCache != null) {
+            try {
+                return profilesCache.loadNamedProfile(this.profile.name);
```

Comment:
Thanks for catching this issue.
In theory, there is one more function that's calling `loadNamedProfile`

CallStack:
`ssoLogin|ssoLogout > getServiceProfile > getServiceProfileForAuthPurposes`

Not sure that we should be super concerned with that other use, since it's using the `cache` 😋 

However, we should warn extenders that may rely on the `loadNamedProfile` 😅